CHANGELOG.md

```diff
@@ -27,11 +28,13 @@
     - E.g. `/indicator?name=Currentness`
 
 [#137]: https://github.com/GIScience/ohsome-quality-analyst/issues/137
+[#170]: https://github.com/GIScience/ohsome-quality-analyst/pull/170
 [#178]: https://github.com/GIScience/ohsome-quality-analyst/pull/178
 [#186]: https://github.com/GIScience/ohsome-quality-analyst/pull/186
 [#190]: https://github.com/GIScience/ohsome-quality-analyst/pull/190
 [#199]: https://github.com/GIScience/ohsome-quality-analyst/pull/199
 [#204]: https://github.com/GIScience/ohsome-quality-analyst/pull/204
+[`scipy`]: https://scipy.org/
```

Comment:
Why is this here?

Reply:
Its from an old sentence in the changelog. I removed this line.